islands/components/DollComponents.tsx

```diff
@@ -1,14 +1,14 @@
-import type { DollTypes } from "../../loaders/MiniMe/minime.ts"
+import type { MiniMe } from "../../loaders/MiniMe/minime.ts";
 import { useRef, useState } from "preact/hooks";
 import Icon from "site/components/ui/Icon.tsx";
 import { ProductDetailsPage } from "apps/commerce/types.ts";
+import { PartiallyEmittedExpression } from "https://deno.land/x/ts_morph@21.0.1/mod.ts";
 
 /**@title Informações da Mini Me*/
 interface Props {
-  dollParts: DollTypes
-  partSelected: DollTypes[];
+  dollParts: MiniMe;
```

Comment:
Use um nome mais lógico. `dollParts` não inclui apenas dados das _Parts_. Use o mesmo nome do tipo.
```suggestion
  minime: MiniMe;
```